Store/pages/StoreNotVisiblePage.php

```diff
@@ -135,9 +135,7 @@ private function getLocaleLink($locale)
 
 		ob_start();
 
-		$a_tag->href = $this->app->getBaseHref(null, $locale->id).
-			$this->source;
-
+		$a_tag->href = $this->app->getBaseHref().$this->source;
```

Comment:
get `getBaseHref` ever take $locale as a property? I looked at `StoreLocaleApplication` and it doesn't and neither does SiteWebApplication.

Reply:
I did the same check. I'm guessing that a long, long time it did - but it no longer does.